SandboxiePlus/SandMan/sandman_de.ts

```diff
@@ -2587,22 +2587,22 @@ Möchten Sie die Bereinigung durchführen?</translation>
     <message>
         <location filename="SandMan.cpp" line="2185"/>
         <source>Select file name</source>
-        <translation type="unfinished"></translation>
+        <translation>Dateinamen auswählen</translation>
     </message>
     <message>
         <location filename="SandMan.cpp" line="2185"/>
         <source>7-zip Archive (*.7z)</source>
-        <translation type="unfinished"></translation>
+        <translation>7-zip Archiv (*.7z)</translation>
     </message>
     <message>
         <location filename="SandMan.cpp" line="2198"/>
         <source>This Name is already in use, please select an alternative box name</source>
-        <translation type="unfinished"></translation>
+        <translation>Dieser Name wird bereits verwendet, bitte wählen Sie einen anderen Boxnamen</translation>
```

Comment:
```suggestion
        <translation>Dieser name wird bereits verwendet, bitte wählen Sie einen anderen Boxnamen</translation>
```
https://github.com/sandboxie-plus/Sandboxie/commit/34c40c86149c0f3e010d48aed8ee1711efebf474